nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1176,6 +1176,148 @@ Secure Prompt
 
 If you prefer not to provide the password or raw key in the command-line invocation of the tool, leaving these arguments absent will prompt a secure console read of the password (by default) or raw key (if the `-r` flag is provided at invocation).
 
+[[admin-toolkit]]
+Administrative Tools
+--------------------
+The admin toolkit contains command line utilities for administrators to support NiFi maintenance in standalone
+and clustered environments. These utilities include:
+
+* Notify -- The notification tool allows administrators to send bulletins to the NiFi UI using the command line.
+* Node Manager -- The node manager tool allows administrators to safely disconnect or remove nodes from a NiFi cluster.
```

Comment:
missing some functionality here. Should read:

Node Manager -- The node manager tool allows administrators to perform a status check on a node as well as connect, disconnect or remove nodes that are part of a cluster.